Lab 1.ipynb

```diff
@@ -0,0 +1,599 @@
cqj+t6H5p6BjIQnOTLJDUnWJVkxzvOvT3Jdkm8l+WKSR3c9d1+Sq5qf1e1GLknqF+eES5pLWh8JTzIPOBt4NjAKrEmyuqqu69rsSmCkqn6e5PeAdwK/1Tz331V1aKtBS5IkST00iJHwZcC6qrqpqu4BzgOO6d6gqi6tqp83i18DFrYcoyRJktQ3gyjC9wFu7loebdZN5CTgH7uW5ydZm+RrSV7YjwAlSZKkfhrEhZkZZ12Nu2Hy28AI8BtdqxdV1YYkjwH+JcnVVfWdcfZdDiwHWLRo0Y5HLUmSJPXIIEbCR4F9u5YXAhvGbpTkCOCNwNFVdffm9VW1ofn3JuAy4LDxOqmqVVU1UlUjCxYs6F30kqS+8I6ZkuaSQRTha4ClSZYk2RU4DnjAt5wkOQz4GzoF+P/P3t3H21WWd/7/fIlCrCgipK0SMokNtaIolEP0Vy312TAY4q/VNmgtKmOGdmKt/XXa2KlAUUtsbX3kV8lgCj4iotVQ0iKjgLZFmyBWCAwaaQaOMEMKGi0WIXLNH3sFt8dzkvOw1977nPN5v177dfa617rvda0AVy7Wude67+pqPzTJQc33w4FnAt0PdEqSZinfjiJpPun7dJSq2pNkHXAFsADYVFXbk5wDbKuqzcCfAQcDH08CcFtVnQI8CTg/yYN0/gdiw5i3qkiSJElDbyCL9VTVFmDLmLYzu74/f4J+/wgc0250kiRJUrtctl6SJEnqM4twSZIkqc8swiVJrUnypCTvS3JpswKyJAmLcEnSBJJsSnJXkhvHtK9MckuSHUnW72uMqrq5qs4AfpXOug+SJCzCJUkTuxBY2d2QZAFwHnAScDRwapKjkxyT5G/GfH6y6XMK8PfAZ/sbviQNr4G8HUWSNPyq6vNJlo5pXgHsaBZMI8nFwOqqOhd48QTjbAY2J7kc+Eh7EUvS7DHjO+FJPpHk5CTeVZekIdTjPH0EcHvX9mjTNtG5n53k3UnOZ8yraccctzbJtiTbdu3a1YMwJWm49eJO+F8CrwbeneTjwIVV9T97MK4kqTd6maczTltNdHBVXQ1cvb9Bq2pjkjuBVQceeODx04xNkmaNGd8Vqar/UVWvAH4e2AlcmeQfk7w6ycNnOr4kaWZ6nKdHgSO7thcDd/QoTpetlzRv9GQKSZLDgFcB/wm4HngXnWR/ZS/GlyTNTA/z9FbgqCTLkhwIrAE29yjGVUk27t69uxfDSdJQ68Wc8E8CXwB+AlhVVadU1ceq6nXAwTMdX5I0M9PN00k+ClwLPDHJaJLTq2oPsA64ArgZuKSqtvciTu+ES5pPejEn/IKq+pGHbZIcVFXfryrfCStJgzetPF1Vp07QvoV9PGQ5XUlWAauWL1/e66Elaej0YjrKW8Zpu7YH40qSemNW5GnvhEuaT6Z9JzzJT9N5LdUjkhzHD5+YfzSdX3lKkgZotuVp74RLmk9mMh3lRXQe8lkM/EVX+3eBP5zBuJKk3phVebqqLgMuGxkZee2gY5Gktk27CK+qi4CLkvxKVX2ihzFJknrAPC1Jw2sm01F+vao+BCxN8rtj91fVX4zTTZLUJ7MtTzsdRcNq6frLWz/Hzg0nt34ODZeZPJj5yObnwcCjxvlIkgZrVuVpH8yUNJ/MZDrK+c3PP+5dOJKkXjFPS9Lwmsl0lHfva39V/fZ0x5Ykzdxsy9NOR5E0n8zk7SjX9SwKSVIbZlWe9u0okuaTmb4dRZI0pMzTkjS8ZjId5Z1V9TtJLgNq7P6qOmUffVcC7wIW0FlOecOY/b8L/CdgD7ALeE1V/a9m32nAHzWHvsW/ZCRpfDPJ05Kkds1kOsoHm59vn0qnJAuA84AXAKPA1iSbq+qmrsOuB0aq6ntJfhP4U+DXkjwWOAsYofMXynVN32/N4Dokaa6aVp4eFOeES5pPpv2Kwqq6rvl5DXAt8C3gHuDapm0iK4AdVXVrVd0PXAysHjP2VVX1vWbzi3RWe4PO6m9XVtU9TeF9JbByutcgSXPZDPL0QPiKQknzyUzeEw5AkpOBbwDvBt4L7Ehy0j66HAHc3rU92rRN5HTgb6faN8naJNuSbNu1a9e+L0KS5rBp5GlJUstmMh1lrz8HnlNVOwCS/AxwOT8snMfKOG0/NlexGevX6Uw9+aWp9q2qjcBGgJGRkXGPkaR5Yqp5WpLUshnfCQfu2pvYG7cCd+3j+FHgyK7txcAdYw9K8nzgvwGnVNX3p9JXkvQjppqnJUktm8nbUX65+bo9yRbgEjp3pV8GbN1H163AUUmWAd8E1gAvHzP2ccD5wMqq6v6L4grgT5Ic2my/EHjjdK9BkuayGeRpSVLLZjIdZVXX9//DD6eM7AIO/fHDO6pqT5J1dArqBcCmqtqe5BxgW1VtBv4MOBj4eBKA26rqlKq6J8mb+eFfHudU1T0zuAZJmsumlacHxbejSJpPZrJYz6tn0HcLsGVM25ld35+/j76bgE3TPbckzRczydOD4IqZkuaTGT+YmWQhnTeYPBlYuLe9ql4z07ElSTNnnpak4dOLBzM/CPw0nXd4X0PnYcnv9mBcSVJvmKclacj0oghfXlVvAu5tlpA/GUTrbsYAACAASURBVDimB+NKknrDPC1JQ6YXRfgDzc9vJ3kKcAiwtAfjSpJ6wzwtSUOmF4v1bGxeGfgmYDOdt5q8qQfjqkeWrr+89XPs3HBy6+eQNG0DzdNJHgl8Hjirqv6mX+eVpGE24yK8qi5ovl4DPGGm40mSemu6eTrJJuDFdBb7eUpX+0rgXXReM3tBVW3Yz1B/QOcd5ZKkRi/ejnIYcDbwTDqLQHwBeHNV3T3TsSVJMzeDPH0h8F7gA11jLQDOA15AZxXjrUk20ynIzx3T/zXAU4Gb6HoriySpN9NRLqbza8ZfabZfAXwMmPBd35KkvppWnq6qzydZOqZ5BbCjqm4FSHIxsLqqzqVz1/xHJHkO8EjgaODfk2ypqgenfymSNDf0ogh/bFW9uWv7LUle0oNxJUm90cs8fQRwe
```

Comment:
You don't need to use k as your x-axis. It appears in your later graphs that part of the distribution is cut off.